drizzle/0011_add_series_update_post.sql

```diff
@@ -0,0 +1,16 @@
+-- Create Series table
+CREATE TABLE IF NOT EXISTS "Series" (
+  "id" SERIAL PRIMARY KEY,
+  "title" TEXT NOT NULL,
+  "description" TEXT,
```

Comment:
I don't think this is used yet so we can delete it for the moment until we have a UI to update this